File: Java Pattern Programs/Symbol Patterns/Pattern 27.java
 class Pattern_113
 {
     public static void main(String a[])
     {
     
     for (int i = 1; i <= 5; i++)
     {
      for (int j = 1;j <= 5;j++)
      {
       if (j == 3 || i == 3)
       {
        System.out.print("* ");
       }
       else
       {
        System.out.print("  ");
       }
      }
      System.out.print("\n");
     }
     
     }
 }

File: CSharpPatternPrograms/Symbol Patterns/Pattern 27.cs
using System;

class Pattern_113
{
    public static void Main(string[] a)
    {

        for (int i = 1; i <= 5; i++)
        {
            for (int j = 1; j <= 5; j++)
            {
                if (j == 3 || i == 3)
                {
                    Console.Write("* "); // space after *
                }
                else
                {
                    Console.Write("  "); // 2 spaces
                }
            }
            Console.WriteLine();
        }
        Console.ReadKey(true);
    }
}
